tests/Microsoft.Identity.Web.Test/WebApiExtensionsTests.cs

```diff
@@ -99,27 +108,6 @@ public void AddProtectedWebApi_WithConfigActions(bool useServiceCollectionExtens
             AddProtectedWebApi_TestCommon(services, provider);
         }
 
-        private void AddProtectedWebApi_TestCommon(IServiceCollection services, ServiceProvider provider)
```

Comment:
I think the private methods should follow after the public ones that use them, in this case. Makes it more readable, IMO.